rclpy/test/test_clock.py

```diff
@@ -23,6 +23,8 @@
 from rclpy.duration import Duration
 from rclpy.time import Time
 
+from .mock_compat import *  # noqa
```

Comment:
Can't do `import .mock_compat` becase [PEP 328](https://www.python.org/dev/peps/pep-0328/) forbids it, and can't do `import mock_compat` because pytest doesn't put the test directory into `sys.path`.

Reply:
Only suppress the specific category rather than a blank `noqa`.